pages/api/feedback/index.ts

```diff
@@ -49,6 +50,13 @@ async function handlePOST(
   const parsedFeedback = createFeedbackSchema.parse(req.body);
   const createdFeedback = await createFeedback(parsedFeedback, userData);
 
+  // Code below should be okay to use void. Await would stop code here and it will not return any kind of error.
+  // So .then or .catch should not be needed
+  void sendFeedbackToGoogleSheets({
+    feedbackText: parsedFeedback.feedbackText,
+    userUUID: userData.uuid,
+  });
+
```

Comment:
Muokkasin `sendFeedbackToGoogleSheets`-funktion olemaan `void`.

Lyhykäisyydessään `await` pysäyttäisi koodin tähän mitä ei haluta ja jos virhe jostain syystä tulisikin sillä ei ole mitään väliä, koska ei sitä tietoa kuitenkaan haluta käyttäjälle lähettää

Reply:
Juuri näin, jos ei käytetä `await` niin silloin pitää lisätä `catch` kuten teit.